torchrl/version.py

```diff
@@ -0,0 +1,2 @@
+__version__ = '0.0.1a0+008bebd'
```

Comment:
So this file was introduced by mistake in a previous PR

It should not be in here